packages/@aws-cdk/aws-apigatewayv2/lib/http/api-mapping.ts

```diff
@@ -0,0 +1,92 @@
+import { Construct, Resource } from '@aws-cdk/core';
+import { CfnApiMapping, CfnApiMappingProps } from '../apigatewayv2.generated';
+import { IApiMapping, IDomainName } from '../common';
+import { IHttpApi } from '../http/api';
+import { IHttpStage } from './stage';
+
+/**
+ * Properties used to create the HttpApiMapping resource
+ */
+export interface HttpApiMappingProps {
+  /**
+   * Api mapping name
+   * @default -  logical id
+   */
+  readonly apiMappingName?: string;
```

Comment:
I don't see this being used anywhere. Drop.

Reply:
Please address this comment.

Reply:
OK fixed.